src/utils/mod.rs

```diff
@@ -6,6 +6,7 @@ use rocket::http::uri::Absolute;
 pub mod cors;
 pub mod context;
 pub mod cache;
+pub mod json;
```

Comment:
Why is this required?

Reply:
Added the `json` utils. At first we had an `Unknown` struct wrapping `serde::Value` to dump all the json, but that proved hard to test (the inner structure was hard to determine from outside)

Reply:
This could be a good place for custom (de)serializers for generic types